client/state/selectors/is-requesting-reader-teams.js

```diff
@@ -0,0 +1,3 @@
+export default function isRequestingReaderTeams( state ) {
+	return !! state.reader.teams.isRequesting;
```

Comment:
I think we all agree that it can be elsewhere.
I think we are going to disagree on how/where to put it somewhere else.

I would be cool with using `lib/inflight`